mavros/src/plugins/sys_status.cpp

```diff
@@ -728,6 +728,7 @@ class SystemStatusPlugin : public plugin::PluginBase
 		state_msg->connected = true;
 		state_msg->armed = !!(hb.base_mode & enum_value(MAV_MODE_FLAG::SAFETY_ARMED));
 		state_msg->guided = !!(hb.base_mode & enum_value(MAV_MODE_FLAG::GUIDED_ENABLED));
+		state_msg->manual = !!(hb.base_mode & enum_value(MAV_MODE_FLAG::MANUAL_INPUT_ENABLED));
```

Comment:
Probably better to keep "manual input" name.

Reply:
Of course! Done in the latest commit.